### DISCUSSION TITLE
This Deployment has been disabled.

### DISCUSSION THREAD

**Discussion by Muhammad-1254** [vercel/community#7339] [Help]
### Summary

I don't know what the problem is. All my hobby projects deployment is disabled, my limit has not been exceeded and I didn't get any email from vercel.
Would anyone be able to help me out with this?
![image](https://github.com/vercel/community/assets/107724827/5efd1e38-1bea-4dc3-a7af-3c882100032c)

### Example

https://muhammad-usman.vercel.app/

### Steps to Reproduce

I wan't to make case study but hobby plan support has moved to our GitHub Discussions community.
![image](https://github.com/vercel/community/assets/107724827/b5d5f5df-3fa3-4e80-8a4d-68e759a32a8a)

**Comment by pawlean** [comment]
Hi @Muhammad-1254!

Sorry to hear that your deployments have been paused. Can you please email us at [abuse@vercel.com](mailto:abuse@vercel.com)?

Thanks! Let me know if I can help with anything else.

[a comment by an automated account is not shown]
